tests/test_modeling_tf_transfo_xl.py

```diff
@@ -97,21 +97,19 @@ def create_and_check_transfo_xl_model(self, config, input_ids_1, input_ids_2, lm
 
         hidden_states_2, mems_2 = model(inputs).to_tuple()
 
-        result = {
-            "hidden_states_1": hidden_states_1.numpy(),
-            "mems_1": [mem.numpy() for mem in mems_1],
-            "hidden_states_2": hidden_states_2.numpy(),
-            "mems_2": [mem.numpy() for mem in mems_2],
-        }
-
-        self.parent.assertEqual(result["hidden_states_1"].shape, (self.batch_size, self.seq_length, self.hidden_size))
-        self.parent.assertEqual(result["hidden_states_2"].shape, (self.batch_size, self.seq_length, self.hidden_size))
+        hidden_states_1 = hidden_states_1.numpy()
+        hidden_states_2 = hidden_states_2.numpy()
+        mems_1 = [mem.numpy() for mem in mems_1]
```

Comment:
I don't think all the calls to numpy are necessary. Do the tests pass without them?

Reply:
It's not needed, indeed. removed and pushed.